src/pytest_cases/case_parametrizer_new.py

```diff
@@ -557,6 +557,12 @@ def get_or_create_case_fixture(case_id,                # type: str
             # Fix the problem with  "case_foo(foo)" leading to the generated fixture having the same name
             existing_fixture_names += fixtures_in_cases_module
 
+    # If the fixture will be injected in a conftest, make sure its name
+    # is unique. Include also its scope to avoid conflicts. See #311.
+    if 'conftest' in target_host.__name__:
```

Comment:
Any reason why this is not an equality check ?
I think that only "conftest.py" named modules are considered by pytest

Reply:
Ok. Then let's include a 1-line comment stating "note that target_host can be "xxxxxx" (where xxxx is a valid relevant example containing conftest and not being trivial conftest.py)